index.html

```diff
@@ -118,27 +165,34 @@ <h2>Main Program</h2>
             <h2 class="feature">Feature speakers</h2>
             <hr class="line">
         </div>
-        <div class="speaker-container">
-            <div class="speaker-card">
-                <div class="images">
-                    <img src="./img/speaker6.jpg" alt="specker"> 
-                </div>
-                <div class="speaker-details">
-                    <div class="info-header">
-                        <h3>bresaola pork</h3>
-                        <p>porchetta drumstick. Prosciutto pancetta chicken shank tail tenderloin </p>
-                        <p class="line-below"></p>
-                    </div>
-                    <div class="info-detail">
-                        <p> porchetta drumstick. Prosciutto pancetta chicken shank tail tenderloin
-                            cyberwarfare</p>
-                    </div>
-                </div>
-            </div>
+        <div class="speaker-container" id="speaker-container">
 
-            <div><button class="speaker-btn" type="button">MORE <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M0 7.33l2.829-2.83 9.175 9.339 9.167-9.339 2.829 2.83-11.996 12.17z"/></svg></button></div>
         </div>
     </section>
+    <section class="partners">
+        <div class="main-program__header">
+            <h2>Partners</h2>
+            <hr class="line">
+        </div>
+        <div class="loggo">
+            <ul>
+                <li><img src="/img/mozilla.png" alt="partners copany"></li>
+                <li><img src="/img/google.png" alt="Google"></li>
+                <li><img src="/img/naver.png" alt="partners copany"></li>
+                <li><img src="/img/daum.png" alt="partners copany"></li>
+                <li><img src="/img/airbnb.png" alt="partners copany"></li>
+            </ul>
+        </div>
+    </section>
+    <footer>
+        <div class="footer__logo">
+            <img src="https://www.dexterindustries.com/wp-content/uploads/2017/01/180px-Scratchcat.svg_.png"
+                alt="splite">
+        </div>
+        <div class="footer__copright">
+            <p>Copyright © 2020 Scratch Rwanda All rights reserved.</p>
+        </div>
+    </footer>
```

Comment:
Great use of semantics 😎 The proper use of tags is key to having better HTML! Well done using section and footer tags 💪 We must not forget to use the `article` tag! Kindly make the necessary changes in your speaker's HTML dynamic creation so that each of them has an `article` tag 🤓